snippets/snippets.json

```diff
@@ -168,6 +168,22 @@
     "prefix": "teste",
     "scope": "source.js"
   },
+  "test.each (table)": {
+    "body": [
```

Comment:
Thanks for your contribution. I'm currently trying the change.

Something looks off. Probably indentation.
![image](https://user-images.githubusercontent.com/13085980/76110092-24066a00-5fde-11ea-9799-c2b81f1a51b7.png)